test/services/GetIncomeSourcesServiceSpec.scala

```diff
@@ -50,12 +50,12 @@ class GetIncomeSourcesServiceSpec extends TestUtils {
           Some(List(SubmittedInterestModel("someName", "123", Some(1234.56), Some(1234.56))))))
 
 
-        (dividendsConnector.getSubmittedDividends(_: String, _: Int, _: String)(_: HeaderCarrier))
-          .expects("12345678", 1234, "87654321", *)
+        (dividendsConnector.getSubmittedDividends(_: String, _: Int)(_: HeaderCarrier))
+          .expects("12345678", 1234, *)
           .returning(Future.successful(expectedDividendsResult))
 
-        (interestConnector.getSubmittedInterest(_: String, _: Int, _: String)(_: HeaderCarrier))
-          .expects("12345678", 1234, "87654321", *)
+        (interestConnector.getSubmittedInterest(_: String, _: Int)(_: HeaderCarrier))
+          .expects("12345678", 1234, *)
```

Comment:
To align with the implementation of the other services and make this more robust we should remove the wildcard `*` from the expects and specify a headerCarrier that contains the extra mtditid header